### PR TITLE
Chore: Hide New Read-Only TAP Admin Field

### PR DESCRIPTION
**Pull request by krisrjohnson21**
This PR adds a new read-only field to the Admin Review stage of the TAP application. This new field prevents Admin from submitting an application without completing the required fields, but only if the application is approved.

### CODE REVIEW

**Comment by krisrjohnson21** [comment]
@karenhawkinson thanks for the guidance! It's been a minute since I had to make CSS changes 🙂 I think it's done correctly now

File: build/stylesheets/admin-tap.css [hunk 1]
```diff
@@ -281,7 +281,9 @@ button.close:active:hover {
 }
 
 /* Hide error field */
-#form-element-wrapper_217 {
+#form-element-wrapper_217,
```
Comment (karenhawkinson) [inline build/stylesheets/admin-tap.css:284]: When I refactored the TAP form, I deleted field 265 thinking it wasn't needed, and ended up replacing it with field 275. Please update the code to match.

#form-element-wrapper_217,
#form-element-wrapper_271,
#form-element-wrapper_275 {
  visibility: hidden;
  height: 0;
  margin: -20px 0;
}

**Review by karenhawkinson — CHANGES_REQUESTED** [review]
@krisrjohnson21 this looks great! One tiny edit to a field number is needed (due to recent my recent changes to the TAP form).

**Review by karenhawkinson — APPROVED** [review]
@krisrjohnson21 looks great!

**Pull request merged**